projects/dashboards/gfbio/src/app/gfbio-collection/gfbio-collection-dialog.component.ts

```diff
@@ -114,7 +114,7 @@ export class GfBioCollectionDialogComponent {
                 const abcdLayerId = layer.id.layerId.substring(abcdIdPos);
                 const numberOfComponents = (abcdLayerId.match(/:/g) || []).length;
 
-                if (numberOfComponents === 4) {
+                if (numberOfComponents > 3) {
```

Comment:
like in the backend:
if there are 4 colons in the identifier: how do you know whether the last part is a unit or part of the dataset name?